sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/DataSourceStrategy.scala

```diff
@@ -320,6 +313,20 @@ class FindDataSourceTable(sparkSession: SparkSession) extends Rule[LogicalPlan]
         v1Relation
       }
   }
+
+  def resolveUnresolvedCatalogRelation(
+      unresolvedCatalogRelation: UnresolvedCatalogRelation): LogicalPlan =
+    unresolvedCatalogRelation match {
```

Comment:
Let's put the method body to {}, to follow conventions of the style guide https://github.com/databricks/scala-style-guide?tab=readme-ov-file#curly-braces

Reply:
Done, thanks